src/app/visual/data-manager/data-manager-visual.component.ts

```diff
@@ -116,7 +121,8 @@ export class DataManagerVisualComponent implements OnInit {
   public settingsKey = 'test';
 
   constructor(
-    private dataManagerService: SkyDataManagerService
+    private dataManagerService: SkyDataManagerService,
+    public themeSvc: SkyThemeService
```

Comment:
Shouldn't this be `@Optional() private themeSvc?: SkyThemeService`?

Reply:
Done